ngraph/test/requirements_test.txt

```diff
@@ -1,8 +1,3 @@
-# PaddlePaddle - generate test models
-paddlepaddle==2.1.0
-gast==0.3.3
-numpy~=1.19.2
-six~=1.15.0
 # ONNX - generate test models
 docopt~=0.6.2
-onnx~=1.9.0
```

Comment:
I believe this can be removed at all since ONNX has its own file now https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/requirements_test_onnx.txt